docs/api.md

```diff
@@ -16,23 +16,27 @@ for invoking function. By default Events API runs on `:4000` port.
 
 ### Event Definition
 
-All data that passes through the Event Gateway is formatted as an Event, based on our default Event schema:
+All data that passes through the Event Gateway is formatted as an CloudEvent, based on CloudEvent schema:
 
-* `event` - `string` - the event name
-* `id` - `string` - the event's instance universally unique ID (provided by the event gateway)
-* `receivedAt` - `number` - the time (milliseconds) when the Event was received by the Event Gateway (provided by the event gateway)
-* `data` - type depends on `dataType` - the event payload
-* `dataType` - `string` - the mime type of `data` payload
+* `event-type` - `string` - the event name
```

Comment:
Let's remove list of fields from here and leave only example and link to original spec.

Reply:
Do you want me to remove the example too or just this list of fields?